controllers/beers.controllers.js

```diff
@@ -0,0 +1,6 @@
+exports.renderBeers = (req, res) => {
+    const beers = punkAPI.getBeers()
+    .then(beersFromApi => console.log('Beers from the database: ', beersFromApi))
+    .catch(error => console.log(error));
+    res.render("beers", { beers });
```

Comment:
You can't render the beers here because ` res.render("beers", { beers });` is outside the `.then()` scope where the beers data is available.

The correct way to pass that data is
```
exports.renderBeers = (req, res) => {
    const beers = punkAPI.getBeers()
    .then(beersFromApi => res.render("beers", { beers });)
    .catch(error => console.log(error));
 
}